sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -845,6 +845,68 @@ class DDLParserSuite extends AnalysisTest {
       ShowTablesStatement(Some(Seq("tbl")), Some("*dog*")))
   }
 
+  test("create namespace") {
+    val sql =
+      """
+        |CREATE DATABASE IF NOT EXISTS database_name
```

Comment:
can we use `a.b.c` instead of `database_name`?

Reply:
updated.